packages/react/src/hooks/actions/useNavigateAction.ts

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { Hub } from '@aws-amplify/core';
+
+import {
+  ACTIONS_CHANNEL,
+  ACTION_NAVIGATE_FINISHED,
+  ACTION_AUTH_SIGNOUT_STARTED,
+  ACTION_NAVIGATE_STARTED,
+} from './constants';
+
+type NavigateType = 'url' | 'anchor' | 'reload';
```

Comment:
Could you export this type as well, just in case anybody wants to accept a `UseNavigateActionProps` externally?

Reply:
Oh yeah, that's a good point. Will do